viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -172,6 +180,20 @@ export class Image360ApiHelper {
     this._needsRedraw = true;
   }
 
+  public remove360ImageCollection(collection: Image360Collection): void {
+    if (
+      this._interactionState.currentImage360Entered !== undefined &&
+      collection.image360Entities.includes(this._interactionState.currentImage360Entered)
+    ) {
+      this.exit360Image();
+    }
+
+    pull(this._imageCollections, collection);
+
+    collection.image360Entities.forEach(entity => this._image360Facade.delete(entity as Image360Entity));
```

Comment:
shouldn't it be `await Promise.all(collection.image360Entities.forEach(entity => this._image360Facade.delete(entity as Image360Entity)));` ?